merlin/table/tensor_table.py

```diff
@@ -44,7 +44,7 @@ def __init__(self, columns: TensorDict = None):
         self._columns = cols_dict
 
     def _convert_arrays_to_columns(self, columns):
-        grouped_columns = group_values_offsets(columns or {})
+        grouped_columns = group_values_offsets(columns if columns is not None else {})
```

Comment:
If it is, it would still get casted to a {} (the idea being it would resolve to None), is the idea that we dont want to be able to pass anything else false-y?

Reply:
No strong opinions here, just wondered what motivated the change. It does potentially better to default things like `[]` to `{}` though.

Reply:
> It does seem potentially better to default things like `[]` to `{}` though.

with this change it apppears we're removing this default coercision. 

Previously `TensorTable([])` would have converted `[]` to `{}` here. With this change `TensorTable([])` raises an AttributeError from within `group_values_offsets`